frontend/packages/kubevirt-plugin/integration-tests/tests/utils/constants/wizard.ts

```diff
@@ -18,7 +19,7 @@ export enum Flavor {
 }
 
 export enum Workload {
-  DESKTOP = 'desktop',
+  DESKTOP = 'Desktop',
```

Comment:
this should be probably changed for all of them, right?

Reply:
right...
fixed and tested